lib/existing-eks-opensource-observability-pattern/index.ts

```diff
@@ -70,6 +70,13 @@ export default class ExistingEksOpenSourceobservabilityPattern {
             );
         }
 
+        if (utils.valueFromContext(scope, "apiserver.pattern.enabled", false)) {
```

Comment:
please structure the `'/../common/resources/amp-config/recording-rules.yml'` to `'/../common/resources/amp-config/api-server/recording-rules.yml'`